wrappers/wasm/CMakeLists.txt

```diff
@@ -11,13 +11,11 @@ if (NOT CMAKE_BUILD_TYPE)
     set (CMAKE_BUILD_TYPE "MinSizeRel" CACHE STRING "Choose the type of build." FORCE)
 endif()
 
-add_definitions ("-s DISABLE_EXCEPTION_CATCHING=0")
```

Comment:
This seems like a duplicate?

Reply:
No, this affecting compilation, the other setting only linking.